install/kubernetes/cilium/values.yaml.tmpl

```
@@ -1652,7 +1665,7 @@ loadBalancer:
     #
     # Applicable values:
     #   - envoy: Enable L7 load balancing via envoy proxy. This will automatically set enable-envoy-config as well.
-    #   - disabled: Disable L7 load balancing.
+    #   - disabled: Disable L7 load balancing via service annotation.
```

No Latin abbreviations
```suggestion
    #   - disabled: Disable L7 load balancing by way of service annotation.
```